org.eclipse.tm4e.ui/src/main/java/org/eclipse/tm4e/ui/internal/utils/ContentTypeHelper.java

```diff
@@ -131,7 +131,10 @@ private static ContentTypeInfo getContentTypes(final ITextFileBuffer buffer) {
 			} catch (final Exception e) {
 				return null;
 			}
-		} catch (final CoreException | IOException ex) {
+		} catch(FileNotFoundException ex) {
```

Comment:
This will not help. The error is not FileNotFoundException but CoreException, so it willl be still logged below

Reply:
OK, I changed it to a catch all but log the exception when tracing is enabled.